README.md

```diff
@@ -248,6 +248,24 @@ You will need to exclude them form `babel-loader`.
 }
 ```
 
+### Top level function (IIFE) is still arrow (on Webpack 5)
+
+That function is defined not by loader, but by Webpack itself. You check or update discussion in [this issue](https://github.com/babel/babel-loader/issues/885). In short - type of the function is defined by `output.environment` in webpack configuration (check more details at [webpack documentation]((https://webpack.js.org/configuration/output/#outputenvironment)). So to have top level function as regular one you need to do next:
```

Comment:
```suggestion
That function is injected by Webpack itself _after_ running `babel-loader`. By default Webpack asumes that your target environment supports some ES2015 features, but you can overwrite this behavior using the `output.environment` Webpack option ([documentation]((https://webpack.js.org/configuration/output/#outputenvironment)).

To avoid the top-level arrow function, you can use `output.environment.arrowFunction`:
```